cdv/cmds/chia_inspect.py

```diff
@@ -125,19 +133,23 @@ def inspect_any_cmd(ctx: click.Context, objects: Tuple[str]):
         for cls in [Coin, CoinSpend, SpendBundle, CoinRecord]:
             try:
                 in_obj = streamable_load(cls, [obj])[0]
+                break
             except Exception:
                 pass
-        # Try it as some key stuff
-        for cls in [G1Element, G2Element, PrivateKey]:
-            try:
-                in_obj = cls.from_bytes(hexstr_to_bytes(obj))  # type: ignore
-            except Exception:
-                pass
-        # Try it as a Program
-        try:
-            in_obj = parse_program(obj)
-        except Exception:
-            pass
+        else:
+            # Try it as some key stuff
+            for cls in [G1Element, G2Element, PrivateKey]:
+                try:
+                    in_obj = cls.from_bytes(hexstr_to_bytes(obj))  # type: ignore
+                    break
+                except Exception:
+                    pass
+            else:
+                # Try it as a Program
+                try:
+                    in_obj = parse_program(obj)
+                except Exception:
+                    pass
```

Comment:
This logic was previously flawed that if it matched something earlier in the code path, it still had a chance to match something later.  This used to not matter because everything had an exact serialization, however, Coin was recently changed so that it would actually parse a coin given a coin record or coin spend object.